exchange/bidder.go

```diff
@@ -164,6 +172,13 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, bidderRequest Bidde
 			if reqInfo.GlobalPrivacyControlHeader == "1" {
 				reqData[i].Headers.Add("Sec-GPC", reqInfo.GlobalPrivacyControlHeader)
 			}
+			if bidRequestMetadata.addCallSignHeader {
+				signatureMessage, err := adCertSigner.Sign(reqData[i].Uri, reqData[i].Body)
+				if err == nil && len(signatureMessage) > 0 {
```

Comment:
I think the idea here is to record this to log or metrics. This will be added in separate PR. I don't think we want to discard the entire request if header was not generated properly. Added a comment.

Reply:
Thanks for adding a comment. Are there any requirements to add a warning in response when debug is enabled to let the publisher know that there was a problem signing the request? If so, is the plan to add that as part of a later PR as well?